frontend/src/metabase/core/components/FormSelect/FormSelect.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useField } from "formik";
+import Select, {
+  SelectOption,
+  SelectProps,
+} from "metabase/core/components/Select";
+
+export interface FormSelectProps<TValue, TOption = SelectOption<TValue>>
+  extends Omit<SelectProps<TValue, TOption>, "value" | "onChange"> {
+  name: string;
+}
+
+function FormSelect<TValue, TOption = SelectOption<TValue>>({
```

Comment:
No `forwardRef` here because `Select` doesn't forward ref as well. We would need to refactor `Select` to a functional component first.